hummingbird/ml/operator_converters/sklearn/pipeline.py

```diff
@@ -0,0 +1,74 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+Converters for operators necessary for supporting scikit-learn Pipelines.
+"""
+
+import numpy as np
+from onnxconverter_common.registration import register_converter
+import torch
+
+from .. import constants
+from .._base_operator import BaseOperator
+
+
+class Concat(BaseOperator, torch.nn.Module):
```

Comment:
I see in `supported.py` you have a comment on "These are ops injected by the parser not "real" sklearn operators."

can you add some other  comment  about that on these two? otherwise they look like new features

Reply:
yep.